Conversion-Class Temperatura

```diff
@@ -0,0 +1,16 @@
+package com.company;
+
+public class Temperatura {
+
+    public double temperatura;
+    public void setTemperatura(double temperaturaZilei) {
+        this.temperatura = temperaturaZilei;
+    }
+    public double getTemperatura(){
+        return temperatura;
+    }
+
+    public void conversionfromFtoC () {
```

Comment:
return a double here, since the conversion should return the value of the conversion